lib/challenge.js

```diff
@@ -51,7 +63,14 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+  name: "Sir Reginald"
+  species: "Canine"
+  breed: "Corgi"
+  makeNoise : [
```

Comment:
The expression assigned to the `makeNoise` property here is an array containing a single object literal, not a function. The object literal is not well-formed, though.